src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -307,7 +307,7 @@ private Timeline2 scrapeQues(BufferedReader br, String url) {
         return quesList;
     }
 
-    private Timeline2 scrapeProfile(BufferedReader br, String url) {
+    public Timeline2 scrapeProfile(BufferedReader br, String url) {
```

Comment:
@vibhcool I have gone through TwitterScraperTest.java. I will update this PR ASAP :)